trillian/db/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mariadb:10.1
```

Comment:
By the way, what is the docker container going to be named, so that I can add it with `FROM`?